### DISCUSSION TITLE
Outline mode?

### DISCUSSION THREAD

**Discussion by simonhf** [alagrede/znote-app#49] [General]
Hello!

I'm evaluating znote and it's looking great so far. Thanks for creating it!

Is there anything like an outline mode for longer markdown documents? And how to enable it?

**Comment by alagrede** [comment]
Hi @simonhf,
I also would like this 😊, Last time I tried to implement this feature I was blocked by the underlying TOC plugin which has an XSS security issue. I will check if it's now solved.
Thanks for the reminder, it will be nice to have a feature. 

Not a high priority but I'll let you know here when it's done.

Anthony

**Comment by alagrede** [comment]
I just released an outline panel to navigate by headers. Hoping this will help you.  🚀
